scripts/pre_commit_linter.py

```diff
@@ -1348,24 +1348,26 @@ def _validate_and_parse_js_files(all_files):
     return parsed_js_files
 
 
-def _get_expression(parsed_node, components_to_check):
+def _get_expression_from_node_if_one_exists(parsed_node, components_to_check):
     """This function returns the expression part of a node.
     Args:
         parsed_node: dict. Parsed node of the body of a JS file.
-        components_to_check: list(str). List of components to check in a
-            JS file.
+        components_to_check: list(str). List of angular components to check
+            in a JS file. These include directives, factories, controllers, etc.
 
     Returns:
         expression: dict. Expression part of the node.
     """
     # Check the type of the node. If the type is
-    # 'ExpressionStatement' then it means the
-    # declaration of a component.
+    # 'ExpressionStatement' (meaning the declaration
+    # of a component) and if not returns None.
     if parsed_node['type'] != 'ExpressionStatement':
         return
-    # Separate the expression part of the node.
+    # Separate the expression part of the node which is the actual
+    # content of the node.
     expression = parsed_node['expression']
-    # Check whether the expression belongs to a component.
+    # Check whether the expression belongs to a component and
+    # if not returns None.
     expression_type_is_not_call = (
```

Comment:
Better to inline this: if expression['type'] != 'CallExpression'.

Use the comment to explain what a CallExpression is and why we require it. The top-level docstring for this function doesn't say anything about CallExpressions so, as a reader, I don't know what they are or why they're relevant.

Reply:
done